QuizApp/Quizzes/View/QuizzesViewController.swift

```diff
@@ -28,73 +27,59 @@ class QuizzesViewController: UIViewController, UITableViewDelegate, UITableViewD
     private var labelFunFact: UILabel!
     private var labelFact: UILabel!
     
-    private var tableView = UITableView()
-    private var quizzes: [Quiz] = []
-    private var sectionedQuizzes = [[Quiz]]()
-    private var noOfNBA = 0
-    
+    private var quizzesPresenter =  QuizzesPresenter()
+    private var QuizTableView = UITableView()
```

Comment:
Generally, CamelCase is used for Type names, not instances, so when you're calling it, it looks like it's a singleton.
It should be `quizTableView`